<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArchiveAccountOptionsWidgetClass</class>
 <widget class="QWidget" name="ArchiveAccountOptionsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>479</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wdtDefault" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblDefault">
        <property name="text">
         <string>Default archiving preferences</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="grlDefault">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbDefaultSave">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lblDefaultSave">
          <property name="text">
           <string>Default archiving rule:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="buddy">
           <cstring>cmbDefaultSave</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdtEntity" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblEntity">
        <property name="text">
         <string>Individual archiving preferences</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vrlEntity">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="tbwItemPrefs">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>JID</string>
           </property>
           <property name="toolTip">
            <string>Contact identifier</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="toolTip">
            <string>Message archiving</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hrlEntityButtons">
          <item>
           <widget class="QLabel" name="lblStatus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbtRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbtAdd</tabstop>
  <tabstop>pbtRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
